docs/docs/0.4.0/policies/fault-injection.md

```diff
@@ -0,0 +1,95 @@
+# Fault Injection
+
+`FaultInjection` policy helps you to test your microservices against resiliency. Kuma provides 3 different types of failures that could be imitated in your environment. 
+These faults are [Delay](#delay), [Abort](#abort) and [ResponseBandwidth](#responsebandwidth-limit) limit.
+
+::: warning
+Please note, `FaultInjection` operates on L7 HTTP traffic, so make sure that selected dataplanes are configured with HTTP Protocol.
+:::
+On Universal:
+
+```yaml
+type: FaultInjection
+mesh: default
+name: fi1
+sources:
+    - match:
+        service: frontend
+        version: "0.1"
+destinations:
+    - match:
+        service: backend
+conf:        
+    abort:
+        httpStatus: 500
+        percentage: 50
+    delay:
+        percentage: 50
+        value: 5s
+    responseBandwidth:
+        limit: 50 mbps
+        percentage: 50    
+```
+
+On Kubernetes:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: FaultInjection
+mesh: default
+metadata:
+  namespace: default
+  name: fi1
+spec:
+    sources:
+        - match:
+            service: frontend
+            version: "0.1"
+            protocol: http
+    destinations:
+        - match:
+            service: backend
+            protocol: http
+    conf:        
+        abort:
+            httpStatus: 500
+            percentage: 50
+        delay:
+            percentage: 50
+            value: 5s
+        responseBandwidth:
+            limit: 50 mbps
+            percentage: 50 
+```
+
+### Sources & Destinations
+`FaultInjection` is a Connection policy, which means it is applied to the connection between dataplanes. As most of the policies, `FaultInjection` supports the powerful mechanism of matching, which allows you to precisely match source and destination services.
```

Comment:
I don't think we use `Connection policy` concept in the docs, so maybe just:
`FaultInjection` is a policy which is applied to the connection between selected dataplanes
?

`source and destination services`
maybe source and destination dataplanes? You can do `service: '*'` and match for other tags